src/main/java/com/byclosure/webcat/stepdefinitions/helpers/SeleniumInteractions.java

```diff
@@ -0,0 +1,192 @@
+package com.byclosure.webcat.stepdefinitions.helpers;
+
+import com.byclosure.webcat.context.IContext;
+import org.openqa.selenium.*;
+import org.openqa.selenium.internal.selenesedriver.TakeScreenshot;
+
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class SeleniumInteractions {
```

Comment:
The point is to use webdriver's methods that do not throw exception when an element is not found (i.e. `driver#findElements`) instead of the ones that throw an Exception when an element is not found (i.e. `driver#findElement`).